src/global_launch/config/globals.yaml

```diff
@@ -13,6 +13,10 @@ mgp_main:
       gps_threshold: 1.5
       force: false
 
```

Comment:
Remove space (PATH nodes should go together)

Reply:
Removed space